Cargo.toml

```diff
@@ -9,24 +9,24 @@ version = "0.1.3"
 edition = "2021"
 
 [workspace.dependencies]
-thiserror = { version = "1.0.63" }
-tokio = { version = "1.40.0", features = ["full"] }
+thiserror = { version = "1.0.64" }
+tokio = { version = "1.38.1", features = ["full"] }
```

Comment:
I'm downgrading `tokio` due to this issue: <https://github.com/open-telemetry/opentelemetry-rust/issues/2094>.

The `tokio` version can be upgrade once the OpenTelemetry library releases the next version that will fix this issue.